modules/installation-gcp-permissions.adoc

```diff
@@ -35,6 +35,9 @@ When you attach the `Owner` role to the service account that you create, you gra
 .Required roles for using the Cloud Credential Operator in passthrough mode
 * Compute Load Balancer Admin
 
+.Required roles for using short-lived credentials
+* IAM Service Account User
```

Comment:
Does this need more explanation? Technically, I think the requirement is anytime users authenticate with credentials that do not include a service account key.

We have docs on how to create a Service Account for GCP: https://docs.openshift.com/container-platform/4.16/installing/installing_gcp/installing-gcp-account.html#installation-gcp-service-account_installing-gcp-account 

(we also allow authentication without service accounts, via Default Application Credentials, which isn't documented). 

I wonder if we should add a note, perhaps in this section, pointing to this. Perhaps something like:

> If the created service account does not include a key, or you are authenticating via short-lived credentials or Default Application credentials, you must include the roles listed in `Required roles for using short-lived credentials`